### PR TITLE
New docs & cleanups

### PR DESCRIPTION
**Pull request by dtbuchholz**
## Summary

Post-hackathon, there was some feedback on missing or outdated docs. This adds a bunch of cleanups, including:
- More in-depth SDK method walkthroughs, covering SDK breaking changes/new options, `node-helpers` jsonFileAliases, full docs on Registry and Validator, adding Nextjs docs, and cleanups for other framework examples.
- SQL examples for many different things (operators, expressions, compound statements).
- Various updates throughout the snippets to try and make the examples more consistent.
- More details smart contract docs for the registry & controller.
- Some callouts on generic access control and how it works.
- Some small style changes for conciseness purposes.
- ORM docs for drizzle and @database/sql.
- Polling controller updates.
- SQL parser addition.

### CODE REVIEW

**Comment by sanderpick** [comment]
very nice! will let joe take a peak

File: docs/playbooks/sql/composing-data.md [hunk 1]
```diff
@@ -110,3 +117,128 @@ ON
...
+## `INTERSECT`
+
+With `INTERSECT`, you can find the intersection across `SELECT` statements. This will return only the rows that are common between the two tables.
+
+```sql
+SELECT
+  id
+FROM
+  my_table
+INTERSECT
```
Comment (joewagner) [inline docs/playbooks/sql/composing-data.md:210]: NRE, This looks good on the deployed site.  I wonder why Github Markdown doesn't highlight it?

File: docs/sdk/README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ The Tableland SDK provides a minimal client that implements the Cloudflare D1 D
...
+The `@tableland/sdk` comes with a few APIs: "primary" [Cloudflare D1 client compatible](https://developers.cloudflare.com/d1/platform/client-api/) API for [`Database`](/sdk/database) connections, and two additional "secondary" APIs ([`Validator`](/sdk/validator) and [`Registry`](/sdk/registry)) that are unique to Tableland operations. Generally, users will leverage the primary database API for typical SQL operations, and the secondary APIs are useful when needing to interact directly with a Tableland Validator node or directly calling Tableland’s smart contracts (on-chain access control, direct contract calls, etc.). Both ESM and CommonJS are supports.
```
Comment (joewagner) [inline docs/sdk/README.md:16]: I am I reading this wrong, or should it be "Both ESM and CommonJS are supported."?

Comment (dtbuchholz) [reply docs/sdk/README.md:16]: ah good catch, typo

File: docs/playbooks/sql/composing-data.md (hunk 1 above)
Comment (dtbuchholz) [reply docs/playbooks/sql/composing-data.md:210]: hm yeah no clue why, odd

**Pull request merged**